### ISSUE TITLE
24H2 OS Build:26100.2605

### ISSUE THREAD

**Issue by xcr89** [lauralex/dwm_lut#69]
Needs an update as of today 12/16 i cant apply LUT anymore with any version due to windows update build.

**Comment by KubaT1987** [comment]
Currently 24H2 is a complete mess. Three days ago - after clean-installing it - I was unable to install Photoshop, with the installation process getting stuck at either 1% or 3%.

There's a good chance DWM LUT will start working correctly once Microsoft fixes all the bugs plaguing 24H2. 

For the time being, your best choice is to stick to 23H2 for a few more months.

**Comment by Ridianoid** [comment]
F.. microsoft ruined everything. I don't know what they're trying, but I don't think this kind of shit has ever been seen in their history. How can they give an update when everything is broken? When they disbanded the Windows update test team, it was obvious that this crap was coming. I'll downgrade to 23h2.

**Comment by casef** [comment]
They made a lot of changes, including (from the stuff that relates to DWM_Lut) improvements of multiplane overlays and introducing the Automatic Color Management.

It's essentially impossible to introduce new stuff without creating potential issues and/or breaking legacy stuff. And it is literally impossible to test the almost infinite configurations that are running Windows in the wild, especially given that a good portion of people having issues that are blamed on Windows are actually user errors, badly maintained systems, systems with straight up hardware issues and so on. Not to mention nVidia issues and/or issues caused by various other drivers and software that are out of Microsoft's hands.

So if you have serious issues with Windows, given that there's a fair amount of us who don't, it's best to start by looking at your own setup and making sure everything is correct on your end before blaming others.

**Comment by CrHasher** [comment]
> They made a lot of changes, including (from the stuff that relates to DWM_Lut) improvements of multiplane overlays and introducing the Automatic Color Management.
> 
> It's essentially impossible to introduce new stuff without creating potential issues and/or breaking legacy stuff. And it is literally impossible to test the almost infinite configurations that are running Windows in the wild, especially given that a good portion of people having issues that are blamed on Windows are actually user errors, badly maintained systems, systems with straight up hardware issues and so on. Not to mention nVidia issues and/or issues caused by various other drivers and software that are out of Microsoft's hands.
> 
> So if you have serious issues with Windows, given that there's a fair amount of us who don't, it's best to start by looking at your own setup and making sure everything is correct on your end before blaming others.

I agree that it's not all Microsoft's fault, but they have shown in recent years that they are fine with firing people and lowering quality. Software used to be tested more thoroughly, and there were many QA personnel. Now, they slash QA to a bare minimum or rely on automated tests that can't yet rival the work of dedicated QA professionals. Additionally, they constantly ship partially finished features, even when it’s obvious that they are not fully complete.
